hack/machineset.sh

```diff
@@ -58,6 +59,7 @@ spec:
         machine.openshift.io/cluster-api-machine-type: worker
         machine.openshift.io/cluster-api-machineset: ${machineSetName}
         machine.openshift.io/os-id: Windows
```

Comment:
Consider removing the `machine.openshift.io/os-id: Windows` label to generate a BYOH MachineSet.

Reply:
The `machine.openshift.io/os-id: Windows` label is used by some machine-api providers as a indicator to perform Windows specific functionality. This label should not be removed even for BYOH MachineSets.

Reply:
Got it.